GenumKit/Parsers/ColorsFileParser.swift

```diff
@@ -66,3 +73,47 @@ public final class ColorsFileParser {
     return value
   }
 }
+
+//MARK: - CLR File Parser
+
+public final class CLRFileParser: ColorsFileParser {
+  public private(set) var colors = [String: UInt32]()
+  
+  public init() {}
+  
+  public func parseFile(path: String) {
+    if let colorsList = NSColorList(name: "UserColors", fromFile: path) {
+      for keyColorPair in colorsList.allKeys.map({ ($0, colorsList.colorWithKey($0)) }) {
+        let colorName = keyColorPair.0.stringByReplacingOccurrencesOfString(" ", withString: "")
+        if let colorValue = keyColorPair.1?.rgbColor?.hexValue {
```

Comment:
Maybe this will make it more readable?

```swift
for nameColorPair in colorsList.allKeys.map(nameColorPair) {
  addColorWithName(nameColorPair.name, value: nameColorPair.value)
}

func nameColorPair(key: String) -> (name: String, value: UInt32)? {
  return colorsList.colorWithKey(key).rgbColor.map({ (key: key.stringByReplacingOccurrencesOfString(" ", withString: ""), color: $0.hexValue) })
}
```